### PR TITLE
address inline comments in various docs

### PR DESCRIPTION
**Pull request by sublimezeal**
### Your checklist for this pull request

Thank you for your contribution to the `next-dev-docs` repo. 

🚨Please review the [Pull request guidelines](../contributor-guide/contributor-guide.md#pull-request-guidelines) to this repository before submitting this PR.

- [X] Your code builds clean without any errors or warnings
  
  ```bash
  yarn build
  ```

- [X] You are using approved terminology

  Refer to the [style guide](../contributor-guide/style-guide.md) for help.

### Description 
1) Resolve yellow-highlighted inline comments in various documents
2) Update sidebars to add "Create Beyond Identity Management API Token" doc and remove placeholder for workforce integration guide

### CODE REVIEW

**Comment by Will5** [comment]
I did a quick read, and overall this looks good to me. I wonder if we can enlist some help to review the integration guides to make sure these are both still accurate and the recommended configurations still make sense

File: docs/integration-guides/auth0.md [hunk 1]
```diff
@@ -38,31 +38,35 @@ Before continuing, make sure you've met the following prerequisites:
...
+   `https://DOMAIN.auth0.com/login/callback`
+
+   where `DOMAIN` is replaced with your Auth0 domain URL
 
      **Example:** 
 
      ```http
-     https://byndid-auth0-demo.us.auth0.com/login/callback
-     ```
+     https://dev-myauth099.us.auth0.com/login/callback
```
[the remark this answers is not in the document]
Comment (sublimezeal) [reply docs/integration-guides/auth0.md:60]: Yes, probably. I had a free trial so I guess those start with 'dev-' (?) then I just anonymized it. But I can use the original name as well.

**Comment by sublimezeal** [comment]
> I did a quick read, and overall this looks good to me. I wonder if we can enlist some help to review the integration guides to make sure these are both still accurate and the recommended configurations still make sense

Yes, I think more can be done for these guides. I'm happy to work with the team on next steps for this.

File: docs/integration-guides/azure-ad-b2c.md [hunk 2]
```diff
@@ -95,10 +95,11 @@ import AddAppAdminConsole  from '../includes/_add-application-console.mdx';
 
 
 ## Configure Azure AD B2C
+In your Azure AD B2C tenant, you'll need to register an application and configure an OIDC provider.
 
-1. [Register an application](https://learn.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=app-reg-ga). <mark>Shouldn't this be a prerequisite?</mark>
+1. First, register a web application following [this tutorial from Microsoft]](https://learn.microsoft.com/en-us/azure/active-directory-b2c/tutorial-register-applications?tabs=app-reg-ga).
```
[the remark this answers is not in the document]
Comment (sublimezeal) [reply docs/integration-guides/azure-ad-b2c.md:100]: thanks!

File: docs/integration-guides/auth0.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sublimezeal) [reply docs/integration-guides/auth0.md:60]: changed it back for now

**Review by turnipdabeets — APPROVED** [review]

**Pull request merged**